pkgs/applications/misc/go-org/default.nix

```diff
@@ -2,13 +2,13 @@
 
 buildGoModule rec {
   pname = "go-org";
-  version = "1.4.0";
+  version = "1.5.0";
 
   src = fetchFromGitHub {
     owner = "niklasfasching";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-nMZzRbu3lxunIlnnmb49Ljt8oSiYpj+8gZ0u/OFRRDM=";
+    sha256 = "sha256-Wp8WEfRcrtn+VdcbehYcOJI5FkPQiyo6nLsTDvR7riE=";
   };
 
   vendorSha256 = "sha256-njx89Ims7GZql8sbVmH/E9gM/ONRWiPRLVs+FzsCSzI=";
```

Comment:
This most likely needs an update, too.

Reply:
I checked by substituting empty value and it still returned the same.

Reply:
Confirmed.